ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -350,6 +478,15 @@ function check_upgrade_status() {
     fi
     echo -e "Upgrade checking start at $(date "+%F %T")\n"
     start_time=$(date "+%s")
+
+    # https://issues.redhat.com//browse/OTA-861
+    # When upgrade is processing, Upgradeable will no set to false
+    if [[ "$(oc get clusterversion version -ojson | jq -r '.status.conditions[] | select(.type == "Upgradeable").status')" != "False" ]]; then
+        echo "As OTA-861 designed, Upgradeable should be set to False when an upgrade is in progress, but actually not"
```

Comment:
> Escalate the info to Error

right.
on a side note, hopefully we can uncomment `return 1` (as you already mentioned) the moment [cvo#1080](https://github.com/openshift/cluster-version-operator/pull/1080) merges. it supposed to merge already, but it is stuck on `Needs acknowledge-critical-fixes-only` label because its quite late in 4.18 cycle..